src/Avalonia.Interactivity/RoutedEvent.cs

```diff
@@ -36,6 +39,12 @@ public class RoutedEvent
             Name = name;
             OwnerType = ownerType;
             RoutingStrategies = routingStrategies;
+            if (!s_globalRoutedEvents.TryGetValue(ownerType, out var eventList))
```

Comment:
I think the events should be added to the global registry when they are registered, not when they are constructed. This is the pattern used for `AvaloniaProperty`s and it makes unit testing easier because there's no automatic static state.